### PR TITLE
feat(launch): initialize genesis validator

### PR DESCRIPTION
**Pull request by lubtd**
Genesis validators are the validators that will be part of the validator set (if sufficient self delegation) when a chain is start. This is a map indexed by the chain id and the address of the validator.

Waiting for #179 to be merged
TODO:

- [x]  Add genesis validation tests

### CODE REVIEW

**Comment by lubtd** [comment]
I will wait on https://github.com/tendermint/spn/pull/195 to finish to get the validate genesis refactoring

File: x/launch/client/cli/query_genesis_validator.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+			}
+			argsAddress, err := cast.ToStringE(args[1])
+			if err != nil {
+				return err
+			}
+
+			params := &types.QueryGetGenesisValidatorRequest{
+				ChainID: argsChainID,
+				Address: argsAddress,
+			}
```
Comment (Pantani) [inline x/launch/client/cli/query_genesis_validator.go:68]:
```suggestion
			params := &types.QueryGetGenesisValidatorRequest{
				ChainID: args[0],
				Address: args[1],
			}
```

**Review by Pantani — DISMISSED** [review]
🖖🏽

File: x/launch/client/cli/query_genesis_validator.go (hunk 1 above)
Comment (lubtd) [reply x/launch/client/cli/query_genesis_validator.go]: That's pretty ugly, we need to address that haha

**Review by Pantani — APPROVED** [review]

**Review by ilgooz — APPROVED** [review]

**Pull request merged**